server/src/QRServer/config.py

```diff
@@ -197,14 +197,20 @@ def __init__(self):
             config=self,
             name='discord.bot.channel_user_notifications.id',
             cli_args=[],
-            description='Discord Channel ID for user notifications such as registration, claiming of account or bans',
+            description='Discord Channel ID for user notifications such as registration or claiming of account',
             default_value='')
         self.discord_bot_max_aliases = ConfigKey(
             config=self,
             name='discord.bot.max_aliases',
             cli_args=[],
             description='Maximum number of aliases per user',
             default_value=1)
+        self.discord_bot_channel_bans_notifications_id = ConfigKey(
```

Comment:
Perhaps the description should indicate that ban channel and user notifications channel can be the same one